src/main/kotlin/ch/wisv/choice/course/model/Study.kt

```diff
@@ -0,0 +1,24 @@
+package ch.wisv.choice.course.model
+
+/**
```

Comment:
We should include a seperate LICENSE file instrad

Reply:
Why not both?

Reply:
That's a legal nightmare if they ever differ. A LICENSE file also apllies to any new files we add in this repository.